jaxrs/src/main/java/feign/jaxrs/JAXRSContract.java

```diff
@@ -57,9 +63,8 @@ protected MethodMetadata parseAndValidateMetadata(Class<?> targetType, Method me
           // Strip off any trailing slashes, since the template has already had slashes appropriately added
           pathValue = pathValue.substring(0, pathValue.length()-1);
       }
-      md.template().insert(0, pathValue);
+      data.template().insert(0, pathValue);
```

Comment:
Mental note do a sweep later to make everything consistent, not sometimes md, data, or metadata!